frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -318,10 +370,25 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     );
   };
 
+  if (_.isEmpty(filteredItems)) {
+    return (
+      <>
+        <EmptyState variant={EmptyStateVariant.full} className="co-status-card__alerts-msg">
+          <Title headingLevel="h5" size="lg">
+            No operators available
+          </Title>
+          <EmptyStateBody>
+            There are no operators that match your operating system and architecture.
```

Comment:
I'd add the expected OS and arch to the message so it's clear what we're expecting

Reply:
Yeah that would be helpful.  I'll update to:

```html
<EmptyStateBody>
  There are no operators that match your operating system ({window.SERVER_FLAGS.GOOS}) 
  and architecture ({window.SERVER_FLAGS.GOARCH}).
</EmptyStateBody>

```

Reply:
I'd take out "your" because it's ambiguous, and you probably need to handle the case where those values aren't set